quickwit/quickwit-search/src/root.rs

```diff
@@ -173,6 +173,7 @@ fn validate_sort_by_field(field_name: &str, schema: &Schema) -> crate::Result<()
 pub(crate) fn validate_request(
     doc_mapper: &dyn DocMapper,
     search_request: &SearchRequest,
+    searcher_context: &SearcherContext,
```

Comment:
could we pass a `&SearcherConfig` instead?

Reply:
fix
https://github.com/quickwit-oss/quickwit/pull/3566/commits/9d02896663ee4181c52422baf6c61cf55b4b29f5